src/common/Platform.js

```diff
@@ -159,5 +159,13 @@
         };
     };
 
+    Math.sign = Math.sign || function (x) {
+        x = +x; // convert to a number
```

Comment:
FYI - this source code came from here:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sign

+x is preferable here for performance.   You want it to fail to NaN sooner rather than in a parse.